pkg/v1/sdk/capabilities/Dockerfile

```diff
@@ -13,6 +13,7 @@ WORKDIR /workspace
 # Copy the Go Modules manifests
 COPY go.mod go.mod
 COPY go.sum go.sum
+COPY apis/ apis/
```

Comment:
Weird.  Are you seeing a failure when doing `make docker-build` because the new `go.mod` files are missing?  I do see such an error.  But why wasn't it a problem for the `apis/cni` and `apis/cpi` directories, which also have `go.mod` files?  I am confused.

Reply:
That's a really good question. I am not super sure on this but my guess is, none of the internal go modules is using `apis/cni` and `apis/cpi` as a direct dependency where as `apis/config` or `apis/cli` might be getting used as direct dependency.

Reply:
Ah yes, that makes sense.  I can see the `api/cli` and `api/config` in the `require` section of the main `go.mod` file but not the other two.  Good deduction 👍